src/advanced/merkelized-list.md

```diff
@@ -108,18 +108,31 @@ Let `T(height, index)` be a value at tree node for element `index` at height
 `height`. Elements `T(0, index)` contain serialized values of the underlying list
 according to [the Exonum binary serialization spec](../architecture/serialization.md).
 Elements `T(height, index)` for `height > 0` are hashes corresponding the following
-rules.
+rules. Each node, including root, hashes with corresponding prefix:
```

Comment:
I think this is somewhat misleading. The root node is not hashed _just_ with the `0x02` prefix; the prefix also includes the number of leaves in the tree (= number of elements in the underlying list). It just makes sense to specify the latter case separately: it follows the same rules regardless of the number of leaves in the tree.